File: Selenium/Activities/Activity15.java
package activities;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;

public class Activity15 {

	public static void main(String[] args) {

		WebDriver driver =  new FirefoxDriver();
		
		driver.get("https://v1.training-support.net/selenium/dynamic-attributes");
		
		System.out.println(driver.getTitle());
		
		driver.findElement(By.xpath("//input[starts-with(@class, 'username-')]")).sendKeys("admin");
		
		driver.findElement(By.cssSelector("input[class^='password-']")).sendKeys("password");
		
		driver.findElement(By.xpath("//button[@type='submit']")).click();
		
		System.out.println(driver.findElement(By.id("action-confirmation")).getText());
		
		driver.close();

	}

}

File: Selenium/Activities/Activity15.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.support.wait import WebDriverWait
from selenium.webdriver.support import expected_conditions

with webdriver.Firefox() as driver:

    wait = WebDriverWait(driver, 10)

    driver.get("https://v1.training-support.net/selenium/dynamic-attributes")

    print(driver.title)

    driver.find_element(By.XPATH, "//input[starts-with(@class, 'username-')]").send_keys("admin")

    driver.find_element(By.CSS_SELECTOR, "input[class^='password-']").send_keys("password")

    driver.find_element(By.XPATH, "//button[@type='submit']").click()

    wait.until(expected_conditions.visibility_of_element_located((By.ID, "action-confirmation")))

    print(driver.find_element(By.ID, "action-confirmation").text)

    driver.close()